src/model/dfcxx/lib/dfcxx/converter.cpp

```diff
@@ -37,7 +37,7 @@ bool DFCIRConverter::convertAndPrint(mlir::ModuleOp module,
   }
   pm.addPass(circt::createLowerFIRRTLToHWPass());
   pm.addPass(circt::createLowerSeqToSVPass());
-  pm.addPass(circt::createExportVerilogPass(out));
+  pm.addPass(circt::createExportVerilogPass(*(outputStreams[SV_OUT_ID])));

```

Comment:
In the future every single access to `outputStreams` will be preceded by an `if`-statement.